persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/UserDAO.kt

```diff
@@ -306,4 +307,5 @@ interface UserDAO {
     suspend fun upsertConnectionStatuses(userStatuses: Map<QualifiedIDEntity, ConnectionEntity.State>)
     suspend fun isAtLeastOneUserATeamMember(userId: List<UserIDEntity>, teamId: String): Boolean
     suspend fun getOneOnOnConversationId(userId: UserIDEntity): QualifiedIDEntity?
+    suspend fun getNameAndHandler(userId: UserIDEntity): NameAndHandleEntity?
```

Comment:
```suggestion
    suspend fun getNameAndHandle(userId: UserIDEntity): NameAndHandleEntity?
```

Maybe we keep as `getNameAndHandle`? without the `r` at the end, otherwise it can look like it will return (despite having the specific return type) a name and a handler class?